src/workerd/api/r2-admin.c++

```diff
@@ -33,9 +40,9 @@ jsg::Promise<jsg::Ref<R2Bucket>> R2Admin::create(jsg::Lock& js, kj::String name,
   createBucketBuilder.setBucket(name);
 
   auto requestJson = json.encode(requestBuilder);
-
+  auto jsonWebToken = jwt.map([](const auto& b) { return kj::str(b); });
   auto promise = doR2HTTPPutRequest(js, kj::mv(client), nullptr, nullptr,
-                                    kj::mv(requestJson), nullptr);
+                                    kj::mv(requestJson), kj::ArrayPtr<kj::StringPtr>(), kj::mv(jsonWebToken));
```

Comment:
It looks like we are only passing the jwt here, but not the "path" (account/bucket). So I don't think these functions would actually work correctly via the new CrossAccount binding.

For the Queues use case, we don't really need the ability to `create` or `list` buckets... but I still don't think its wise to leave it in this half-implemented state. Probably best to go ahead and plumb the "path" through here.